libr/bin/format/elf/elf.c

```diff
@@ -2347,6 +2347,7 @@ void* Elf_(r_bin_elf_free)(struct Elf_(r_bin_elf_obj_t)* bin) {
 	return NULL;
 }
 
+//XXX: this function will break unless r_file_slurp is fixed
```

Comment:
Sorry, I forgot about it. The issue is that for large file, r_file_slurp would try to get whole file at a time into the memory, but for large files, it would fail. What I require for the code to run is RIOBind with `RIO != NULL`. Though I have checked at all the places for RIOBind and RIO, in case they are absent, it will work as before (fail for large files).